src/podofo/CMakeLists.txt

```diff
@@ -301,9 +301,9 @@ IF(PODOFO_BUILD_SHARED)
         CACHE INTERNAL "Which PoDoFo library variant to depend on")
     SET(USING_SHARED_PODOFO TRUE)
     INSTALL(TARGETS podofo_shared
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "${LIBDIRNAME}"
-        ARCHIVE DESTINATION "${LIBDIRNAME}"
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
I verified even if this variable contains spaces, it won't break the install:

`cmake -DCMAKE_INSTALL_LIBDIR='libs n shit'` works as expected, no surprises. :-)

Reply:
Yes, that's magic of CMake